.gitignore

```
@@ -71,3 +69,6 @@ temp/
 
 # scan temporary files
 **/fastlane/test_output
+
+# FVM Version Cache
+.fvm/
```

```suggestion
.fvm/

```
Let's keep the newline at the end.